src/aiortc/rtcicetransport.py

```diff
@@ -136,6 +136,7 @@ def connection_kwargs(servers: List[RTCIceServer]) -> Dict[str, Any]:
 
 def parse_stun_turn_uri(uri: str) -> Dict[str, Any]:
     if uri.startswith("stun"):
+        uri = uri.partition("?")[0].rstrip()
```

Comment:
I'm not sure we want rstrip(), there really is no reason to have whitespace in a URL